libraries/AP_HAL_ChibiOS/hwdef/CSKY405/Readme.md

```diff
@@ -0,0 +1,100 @@
+# CSKY405 Flight Controller
+
+The CSKY405 is a flight controller produced by [ClearSky](http://csky.space/).
+
+## Features
+    Processor
+        STM32F405 168Mhz, 1MB 32-bit processor
+        AT7456E OSD
+    Sensors
+        BMI088 Acc/Gyro
+        BMP390 barometer
+    Power
+        2S - 6S Lipo input voltage with voltage monitoring
+        90A Cont., 215A peak current monitor
+        12V, 2A BEC for powering Video Transmitter controlled by GPIO
+        5V, 2.5A BEC for internal and peripherals
+    Interfaces
+        10x PWM outputs DShot capable
+        1x RC input
+        6x UARTs/serial for GPS and other peripherals
+        I2C port for external compass, airspeed, etc.
+        microSDCard for logging, etc.
+        USB-C port
+  
+
+## Overview
+
+![CSKY405](CSKY405.png)
+
+## Wiring Diagram
+
+![CSKY405 Wiring](CSKY405_wiring.png)
+
+## UART Mapping
+
+The UARTs are marked Rn and Tn in the above pinouts. The Rn pin is the
+receive pin for UARTn. The Tn pin is the transmit pin for UARTn.
+
+ - SERIAL0 -> USB
+ - SERIAL1 -> UART4  (MAVLink2 telem)
+ - SERIAL2 -> USART1 (Serial RC input) (DMA capable)
+ - SERIAL3 -> UART5  (GPS)
+ - SERIAL4 -> USART3 (User) (DMA capable)
+ - SERIAL5 -> USART6 (User) (TX DMA capable)
+ - SERIAL6 -> USART2 (RX tied to inverted SBUS RC input, but can be used as normal UART if :ref:`BRD_ALT_CONFIG<>` =1) 
+
+
+## RC Input
+
+RC input is configured on the SBUS pin (inverted and sent to UART2_RX). It supports all RC
```

Comment:
I cant seem to find the SBUS pin on picture...and your wiring diagram show an SBUS Receivers tied to RX2 which does not work

Reply:
there is really no marking of the SBUS pin, instead there is an RX2 to which an S BUS can be connected (STM32F405 can invert this pin using the USART peripherals) and it works, also the RX2 pin can work like a regular USART